code/modules/atmospherics/gasmixtures/gas_types.dm

```diff
@@ -1,4 +1,5 @@
 GLOBAL_LIST_INIT(hardcoded_gases, list(/datum/gas/oxygen, /datum/gas/nitrogen, /datum/gas/carbon_dioxide, /datum/gas/plasma)) //the main four gases, which were at one time hardcoded
+GLOBAL_LIST_INIT(nonreactive_gases, list(/datum/gas/oxygen, /datum/gas/nitrogen, /datum/gas/carbon_dioxide)) // These gasses cannot react amongst themselves
```

Comment:
Add to the list: `/datum/gas/nitryl`, `/datum/gas/bz`, `/datum/gas/stimulum` and `/datum/gas/pluoxium`.

Reply:
I'll try it out later - its a performance question for me but I'm skeptical that this would help. 

While you're technically correct that those items will also not react, they're also exceptionally rare. But then again ANY reaction is pretty uncommon, but then agaaaaain those gases are far more likely to be found in a reactive gas environment where this check is pointless. So basically:

- If a reaction needs to happen, checking for nitryl/bz/stimulum/pluox will slow things down.
but
- If you have isolated the nitryl/bz/stimulum/pluox in a pipenet (i.e. behind a filter) then it speeds things up again.

Reply:
Actually no, you loop through gases in the turf not through the typecache